tests/javaobject/cli/InstantiateDataSerializable.java

```diff
@@ -32,15 +32,16 @@
 public class InstantiateDataSerializable extends FunctionAdapter implements Declarable{
 
 public void execute(FunctionContext context) {
-  Instantiator.register(new Instantiator(javaobject.cli.Position.class, 22) {
+
+  Instantiator.register(new Instantiator(javaobject.cli.PositionKey.class, 21) {
     public DataSerializable newInstance() {
-      return new javaobject.cli.Position();
+      return new javaobject.cli.PositionKey();
     }
   });
 
-  Instantiator.register(new Instantiator(javaobject.cli.PositionKey.class, 77) {
```

Comment:
Why change the ID from 77 to 21? Have all the older tests expecting 77 been updated?

Reply:
Yes. All tests updated and pass. I had to use something other than 77, as it was causing a conflict (don't remember where). Chose 21 for PositionKey since it was unused and close to Position (22).